src/sidebar/store/modules/test/selection-test.js

```diff
@@ -201,6 +201,20 @@ describe('sidebar/store/modules/selection', () => {
     });
   });
 
+  describe('changeFocusModeUser()', function() {
```

Comment:
Do we need tests for the sad path here? What happens if an improperly-formatted `user` object is sent as a method param? What if one of the properties is missing, e.g.?

Reply:
I took a close look at this, yes we're setting the user here but the code that checks the user later on is tested in other functions on line 264 and 271